core/src/test/java/com/sequenceiq/cloudbreak/controller/v4/StackV4ControllerTest.java

```diff
@@ -67,4 +75,10 @@ void generateImageCatalogInternalTest() {
 
         verify(stackOperations).generateImageCatalog(NameOrCrn.ofName(STACK_NAME), WORKSPACE_ID);
     }
+
+    @Test
+    public void testCcmUpgrade() {
+        when(stackCcmUpgradeService.upgradeCcm(any(), any())).thenReturn(new FlowIdentifier(FlowType.FLOW, "1"));
+        ThreadBasedUserCrnProvider.doAs(USER_CRN, () -> underTest.upgradeCcmByNameInternal(STACK_NAME, USER_CRN));
```

Comment:
Please verify the return value of `underTest.upgradeCcmByNameInternal()` that it matches the `FlowIdentifier` instance created above.